source/Engine.cpp

```diff
@@ -130,9 +130,19 @@ namespace {
 		Messages::Add(tag + message, Messages::Importance::High);
 	}
 
+	double FlareCurve(double x)
+	{
+		return x * x * x;
+	}
+
 	void DrawFlareSprites(const Ship &ship, DrawList &draw, const vector<Ship::EnginePoint> &enginePoints,
-		const vector<pair<Body, int>> &flareSprites, uint8_t side)
+		const vector<pair<Body, int>> &flareSprites, uint8_t side, Point scale)
 	{
+		const auto maxHeldReciprocal = 1. / Ship::MAX_THRUST_HELD_FRAMES;
+		Point leftTurnScale = scale * Point(ship.ThrustHeldFrames(Ship::ThrustKind::LEFT) * maxHeldReciprocal,
+			FlareCurve(ship.ThrustHeldFrames(Ship::ThrustKind::LEFT) * maxHeldReciprocal));
+		Point rightTurnScale = scale * Point(ship.ThrustHeldFrames(Ship::ThrustKind::RIGHT) * maxHeldReciprocal,
+			FlareCurve(ship.ThrustHeldFrames(Ship::ThrustKind::RIGHT) * maxHeldReciprocal));
```

Comment:
```suggestion
		Point leftTurnScale = scale * Point(ship.ThrustHeldFrames(Ship::ThrustKind::LEFT) * Ship::THRUST_HELD_FRAMES_RECIP,
			FlareCurve(ship.ThrustHeldFrames(Ship::ThrustKind::LEFT) * Ship::THRUST_HELD_FRAMES_RECIP));
		Point rightTurnScale = scale * Point(ship.ThrustHeldFrames(Ship::ThrustKind::RIGHT) * Ship::THRUST_HELD_FRAMES_RECIP,
			FlareCurve(ship.ThrustHeldFrames(Ship::ThrustKind::RIGHT) * Ship::THRUST_HELD_FRAMES_RECIP));
```
Should still fit within the line length limit, but adjust if you want.